main.py

```
@@ -1538,37 +1538,47 @@ async def callback_router(update: Update, context: ContextTypes.DEFAULT_TYPE):
+                    text=dashboard_text,
+                    parse_mode="HTML",
+                    reply_markup=reply_markup
+                )
+            else:
+                await context.bot.send_message(
+                    chat_id=chat_id,
+                    text=dashboard_text,
+                    parse_mode="HTML",
+                    reply_markup=reply_markup
+                )
+        except Exception as e: logging.error(f"Dashboard update error: {e}")
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)